src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import TodoList from './ToDo/TodoList';
+import Context from './context';
+import AddTodo from './ToDo/AddTodo';
+import axios from 'axios';
+
+function App() {
+  const [todos, setTodos] = useState([])
+  todos.sort((a, b) => (a.isCheck > b.isCheck) ? 1 : a.isCheck < b.isCheck ? -1 : 0);
+  useEffect(() => {
+    fetchPost()
+  })
+
+  async function fetchPost() {
+    const response = await axios.get('http://localhost:8000/tasks')
```

Comment:
axios.get('http://localhost:8000/tasks')
Take out all requests in services. This is a separate folder, it contains a file with all requests

Reply:
http://localhost:8000 
take it out to a constant